firebase/README.md

```diff
@@ -18,3 +18,21 @@ Now you are inside the docker container and can login to firebase. You need to i
 Finally you can deploy your changes for cloud functions and database rules individually.
 * `firebase deploy --only functions`
 * `firebase deploy --only database:rules`
+
+## Notes on OAuth (OSM login)
+
+The main notes are in the app repository (/docs/osm_login.md).
+
+Some specifics about the related functions:
+ - get a service-account.json file from firebase which allows the OAuth functions to access the database and call
```

Comment:
I used a separate one while testing to avoid messing things up. I think we should be able to reuse the current one, we just need to double check it for permissions. I think this just needs executing functions and accessing the database (and opening non-firebase urls, but we get that from being on the blaze plan, apparently).